bin/audio-offset-finder

```diff
@@ -29,12 +30,17 @@ def main():
     parser.add_argument('--sr', metavar='sample rate', type=int, default=8000, help='Target sample rate during downsampling')
     parser.add_argument('--trim', metavar='seconds', type=int, default=60*15, help='Only uses first n seconds of audio files')
     parser.add_argument('--resolution', metavar='samples', type=int, default=128, help='Resolution (maximum accuracy) of search in samples')
+    parser.add_argument('--plot', action='store_true', dest='plot', help='Display plot of cross-correlation results')
     args = parser.parse_args()
     if not (args.find_offset_of or args.within):
         parser.error("Please input audio files")
-    offset, score = find_offset(args.within, args.find_offset_of, fs=int(args.sr), trim=int(args.trim), hop_length=int(args.resolution))
-    print("Offset: %s (seconds)" % str(offset))
-    print("Standard score: %s" % str(score))
+    results = find_offset(args.within, args.find_offset_of, fs=int(args.sr), trim=int(args.trim), hop_length=int(args.resolution))
+    print("Offset: %s (seconds)" % str(results["offset"]))
+    print("Standard score: %s" % str(results["score"]))
+    if args.plot:
+        pyplot.figure(figsize=(15, 5))
+        pyplot.plot(results["correlation"])
+        pyplot.show()
```

Comment:
Is it possible to save the plot to a file, such as PNG? I use WSL2 on Windows 10 and get this error:

> Matplotlib is currently using agg, which is a non-GUI backend, so cannot show the figure.

Reply:
As of [this commit](https://github.com/bbc/audio-offset-finder/pull/18/commits/bba5447e1116a4d285a34cfabfbdff77d2bd0d4e) the --plot argument has been split: --show-plot will show the plot on-screen if the matplotlib backend supports that; --save-plot will save it to a file.  A basic unit test for the latter will follow in a future PR.